docs/docs/glossary/progressive-enhancement.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Progressive Enhancement
+disableTableOfContents: true
+---
+
+Learn what _progressive enhancement_ is and how Gatsby it easy to build sites using progressive enhancement principles.
```

Comment:
Please avoid the word "easy" here. I think there's also a word missing. "makes"?
```suggestion
Learn what _progressive enhancement_ is and how Gatsby builds sites using progressive enhancement principles by default.
```